pkg/sdkserver/sdkserver.go

```diff
@@ -107,6 +111,9 @@ func NewSDKServer(gameServerName, namespace string, kubeClient kubernetes.Interf
 		healthMutex:        sync.RWMutex{},
 		healthFailureCount: 0,
 		streamMutex:        sync.RWMutex{},
+		gsLabels:           map[string]string{},
+		gsAnnotations:      map[string]string{},
+		gsUpdateMutex:      sync.RWMutex{},
```

Comment:
nit: no need to initialize mutex like that (default value is fine). feel free to remove all of those.